webshare/share-securecontext.http.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
```

Comment:
`http.html` does not mean anything. As @foolip noted, tests run on a non-secure origin by default.

On your local machine, you should be setting `web-platform.test` and a few other domains to point to loopback, as outlined in https://github.com/w3c/web-platform-tests#running-the-tests. If you do that, then loading `http://web-platform.test:8000/` locally should do the trick.